src/neural/loader.cc

```diff
@@ -33,6 +33,8 @@
 
 namespace lczero {
 
+static const std::uint32_t kWeightMagic = 0x1c0;
```

Comment:
C++ way is to put it into anonymous namespace (two lines below).
Then `static` is not really needed.